testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/ResetPasswordTest.java

```diff
@@ -1293,7 +1293,11 @@ private void resetPasswordTwiceInNewTab(UserRepresentation user, String clientId
             logoutConfirmPage.assertCurrent();
             logoutConfirmPage.confirmLogout();
 
-            events.expectLogout(sessionId).user(user.getId()).removeDetail(Details.REDIRECT_URI).assertEvent();
+            events.expectLogout(sessionId)
```

Comment:
Removing the `null` client in `AssertEvents` forced me to add a real `.client` here to not fail the check.